common/TruffleConfig.js

```diff
@@ -184,7 +184,7 @@ const TruffleConfig = {
   },
   compilers: {
     solc: {
-      version: "0.6.6",
+      version: "0.6.12",
```

Comment:
Does this mean we need to change our contract's pragma version?

Reply:
It shouldn't if I remember correctly. I think the pragmas only care about the `0.6` part, not the patch part of the semver string.